lib/services.template.ejs

```diff
@@ -20,8 +20,7 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
     var m = url.match(/^(?:https?:)?\/\/([^\/]+)/);
     return m ? m[1] : null;
   }
-
-  var urlBaseHost = getHost(urlBase) || location.host;
+  var urlBaseHost = getHost(urlBase) ? urlBase : location.host;
```

Comment:
I think this is modifying the behaviour in a very subtle way, because when `getHost(urlBase)` returns truthy value, then the original `urlBase` is used. Is that intentional?